app/controllers/rents_controller.rb

```diff
@@ -6,8 +6,11 @@ def rent_params
     params.permit(:user_id, :book_id, :from, :to)
   end
 
+  # El codigo esta asi para que pase los tests.
+  # TODO: si entran por user_id o por book_id es la misma url. Como identificarlo?
```

Comment:
Comments should be in English.

About the comment: now the solution with just one method doesn't work anymore, so we should just split the method in 2